README.md

```diff
@@ -27,9 +30,17 @@ API Definition
 --------------
 
 See the human-readable [Reference Documentation](https://ga4gh.github.io/task-execution-schemas/docs/)
-and the [OpenAPI YAML description](openapi/task_execution_service.openapi.yaml). You can also explore the specification in the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml).
+You can also explore the specification in
+the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml)**
+*Manually load the JSON if working from a non-develop branch version.* Preview documentation from
+the [gh-openapi-docs](https://github.com/ga4gh/gh-openapi-docs) for the development
+branch [here](https://ga4gh.github.io/task-execution-schemas/preview/develop/docs/index.html)
+
+> All documentation and pages hosted at 'ga4gh.github.io/task-execution-schemas' reflect the latest API release from
+> the `master` branch. To monitor the latest development work, add 'preview/\<branch\>' to the URLs above (e.g., '
+> ga4gh.github.io/task-execution-schemas/preview/\<branch\>/docs'). To view the latest *stable*
+> development API specification, refer to the `develop` branch.
```

Comment:
I can't comment on this, but I think the section "TES Compliant Implementations" has some issues:
- I don't think it makes sense to use note/quote markdown for an entire section. Rather, I would emphasize (in bold) the most important bit or bits.
- The wording is strange and there is a comma missing.

How about:

> Complying with GA4GH policies and regulations, security reviews are performed for each major version release of the API. However, **no security guarantees are given for any implementation of the API, including any linked to from this page or the accompanying documentation**. Use such implementations at your own risk and make sure to arrange for a security audit of your application to ensure compliance with any applicable regulatory and security requirements, especially where personal data may be at issue.

Reply:
Good idea. Better workind